// Location messages for udaconnect app
// Compile with:
// python3 -m grpc_tools.protoc -I./ --python_out=./ --grpc_python_out=./ location.proto
syntax = "proto3";

message Location {
  
  int32 id = 1;
  int32 person_id = 2;
  float longitude = 3;
  float latitude = 4;
  float creation_time = 5;

}

message LocationRequest {

  int32 id = 1;

}

service LocationService {
  rpc Create(Location) returns (Location);
  rpc Get(LocationRequest) returns (Location);
}
